### PR TITLE
deploying-image.rst: Include bmap instructions

### PR DESCRIPTION
**Pull request by fhajredini**
Signed-off-by: Fisnik Hajredini <fhajredini@luxoft.com>

### CODE REVIEW

File: docs/articles/baseplatform/deploying-image.rst [hunk 1]
```diff
@@ -6,6 +6,33 @@ Deploying the image
 This section describes how to deploy the build output, a ready disk-image, to a removable media.
 Further information can be found in the `Poky documentation`_.
 
+Recommended instructions
+------------------------
+
+Use bmap-tools to write an Intel image to a disk. You will need two files, both of them will be both found on ``/tmp/deploy/images/<architecture>/``.
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: both both

File: docs/articles/baseplatform/deploying-image.rst [hunk 2]
```diff
@@ -6,6 +6,33 @@ Deploying the image
 This section describes how to deploy the build output, a ready disk-image, to a removable media.
 Further information can be found in the `Poky documentation`_.
 
+Recommended instructions
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: why?

File: docs/articles/baseplatform/deploying-image.rst [hunk 3]
```diff
@@ -6,6 +6,33 @@ Deploying the image
...
+------------------------
+
+Use bmap-tools to write an Intel image to a disk. You will need two files, both of them will be both found on ``/tmp/deploy/images/<architecture>/``.
+
+The image name and bmap file will have these suffixes:
+
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst:17]: too many empty lines

File: docs/articles/baseplatform/deploying-image.rst [hunk 4]
```diff
@@ -6,6 +6,33 @@ Deploying the image
...
+
+- *.rootfs.wic
+- *.rootfs.wic.bmap 
+
+You will need to install the bmap tools.
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: just add prerequisites section

File: docs/articles/baseplatform/deploying-image.rst (hunk 2 above)
Comment (fhajredini) [reply docs/articles/baseplatform/deploying-image.rst]: Its faster than dd-ing?

Comment (sashko) [reply docs/articles/baseplatform/deploying-image.rst]: are you asking me?

Comment (fhajredini) [reply docs/articles/baseplatform/deploying-image.rst]: Im telling you.. its more than twice faster.

Comment (sashko) [reply docs/articles/baseplatform/deploying-image.rst]: Tell this to a reader.

File: docs/articles/baseplatform/deploying-image.rst [hunk 5]
```diff
@@ -6,6 +6,34 @@ Deploying the image
 This section describes how to deploy the build output, a ready disk-image, to a removable media.
 Further information can be found in the `Poky documentation`_.
 
+Recommended instructions
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: Call it `Using bmap-tool` or something instead

File: docs/articles/baseplatform/deploying-image.rst [hunk 6]
```diff
@@ -6,6 +6,34 @@ Deploying the image
...
+------------------------
+
+Use bmap-tools to write an Intel image to a disk more than twice faster than ``dd``. You will need two files, both of them will be found on ``/tmp/deploy/images/<architecture>/``.
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: length

File: docs/articles/baseplatform/deploying-image.rst [hunk 7]
```diff
@@ -6,6 +6,34 @@ Deploying the image
...
+
+The image name and bmap file will have these suffixes:
+
+- <image>.rootfs.wic
+- <image>.rootfs.wic.bmap 
+
+Now for flashing:
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: To flash the image, run

File: docs/articles/baseplatform/deploying-image.rst [hunk 8]
```diff
@@ -6,6 +6,34 @@ Deploying the image
...
+
+.. code-block:: bash
+
+    sudo bmaptool copy --bmap <bmap-filename> <image-name> <host-device>
+
+.. note:: In case bmap gives you the error "[Errno 16] Device or resource
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: don't use `you` so extensively

File: docs/articles/baseplatform/deploying-image.rst [hunk 9]
```diff
@@ -6,6 +6,34 @@ Deploying the image
...
+          busy", then you have to make sure to unmount the partitions of the disk you are
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst]: make sure the disk is unmounted

**Comment by fhajredini** [comment]
@sashko

File: docs/articles/baseplatform/deploying-image.rst [hunk 10]
```diff
@@ -6,6 +6,35 @@ Deploying the image
 This section describes how to deploy the build output, a ready disk-image, to a removable media.
 Further information can be found in the `Poky documentation`_.
 
+Using bmap-tools
+------------------------
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst:10]: -----

File: docs/articles/baseplatform/deploying-image.rst [hunk 11]
```diff
@@ -6,6 +6,35 @@ Deploying the image
...
+To flash the image, run:
+
+.. code-block:: bash
+
+    sudo bmaptool copy --bmap <bmap-filename> <image-name> <host-device>
+
+.. note:: In case bmap gives the error "[Errno 16] Device or resource
+          busy", make sure the disk partitions are unmounted. 
+
+Prerequisites
```
Comment (sashko) [inline docs/articles/baseplatform/deploying-image.rst:30]: move this section on the top

**Review by sashko — COMMENTED** [review]
@fhajredini why have you merged this without fixing a single issue I have pointed into?

**Pull request merged**